packages/pwa/app/components/forms/useCreditCardFields.jsx

```diff
@@ -8,16 +8,31 @@ import cardValidator from 'card-validator'
 import {useIntl, defineMessages} from 'react-intl'
 
 const messages = defineMessages({
-    required: {defaultMessage: 'Required'},
-    cardNumberInvalid: {defaultMessage: 'Please enter a valid card number'},
-    nameInvalid: {defaultMessage: 'Please enter a valid name'},
-    dateInvalid: {defaultMessage: 'Please enter a valid date'},
-    codeInvalid: {defaultMessage: 'Please enter a valid security code'},
-    cardNumber: {defaultMessage: 'Card Number'},
-    cardType: {defaultMessage: 'Card Type'},
-    cardName: {defaultMessage: 'Name on Card'},
-    expiryDate: {defaultMessage: 'Expiry Date'},
-    securityCode: {defaultMessage: 'Security Code'}
+    required: {defaultMessage: 'Required', id: 'use_credit_card_fields.error.required'},
```

Comment:
I thought I've added the ids for this component in my previous PR. 🤔 . Why are we seeing it here?